definitions/artifacts/azure-databricks-workspace.json

```diff
@@ -0,0 +1,116 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema",
+	"$id": "https://schemas.massdriver.cloud/definitions/artifacts/azure-databricks-workspace.json",
+	"$md": {
+		"access": "public"
+	},
+	"type": "object",
+	"title": "Azure Databricks Workspace",
+	"description": "",
+	"additionalProperties": false,
+	"required": [
+		"data",
+		"specs"
+	],
+	"properties": {
+		"data": {
+			"title": "Artifact Data",
+			"type": "object",
+			"required": [
+				"infrastructure",
+				"security"
+			],
+			"properties": {
+				"infrastructure": {
+					"title": "Infrastructure configuration",
+					"type": "object",
+					"required": [
+						"ari",
+						"url",
+						"storage_account_name",
+						"raw_container",
+						"scratch_container",
+						"enriched_container",
+						"application_id"
+					],
+					"properties": {
+						"ari": {
+							"$ref": "../types/azure-resource-id.json"
+						},
+						"url": {
+							"title": "Azure Databricks workspace URL",
+							"$ref": "../types/https-url.json"
+						},
+						"storage_account_name": {
+							"title": "Storage Account Name",
+							"type": "string",
+							"description": "The name of the storage account where the raw, scratch, and enriched data will be stored in separate containers."
+						},
+						"raw_container": {
+							"title": "Raw Container",
+							"description": "The name of the container where the raw data will be stored.",
+							"$ref": "../types/azure-infrastructure-storage-container.json"
+						},
+						"scratch_container": {
+							"title": "Scratch Container",
+							"description": "The name of the container where intermediate data transformations or experimental analyst data will be stored.",
+							"$ref": "../types/azure-infrastructure-storage-container.json"
+						},
+						"enriched_container": {
+							"title": "Enriched Container",
+							"description": "The name of the container where curated data assets will be stored.",
+							"$ref": "../types/azure-infrastructure-storage-container.json"
+						},
+						"application_id": {
```

Comment:
Applications are IAM entities in Azure IAM, right?  With our security model, the downstream resource would generally create it's own IAM entity and assign on or more of the roles specified in the security block to give itself the required accesses.  If we're needing to forward IAM **entity** information that seems like we might not have appropriate boundaries set up in the bundle.

Reply:
hmm the reason I created an an azure application entity in this bundle is in order to the mount to the data lake storage to any databricks clusters that are spawned in this workspace. I think you're right that this app id probably does not need to be forwarded to downstream deps. I will remove.